### PR TITLE
Native ads Aspect Ratio Support.

### PR DESCRIPTION
**Pull request by ManojBudumuru-vungle**
This commit will add Native ads aspectRatio support for adapter.

IOS-7016

### CODE REVIEW

**Review by stanley-vungle — APPROVED** [review]

File: adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleNativeAd.m [hunk 1]
```diff
@@ -178,6 +178,13 @@ - (BOOL)hasVideoContent {
   return YES;
 }
 
+- (CGFloat)mediaContentAspectRatio {
+    if (_nativeAd != nil) {
```
Comment (stanley-vungle) [inline adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleNativeAd.m:182]: You can just do `if (_nativeAd)`

File: adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleNativeAd.m [hunk 2]
```diff
@@ -178,6 +178,13 @@ - (BOOL)hasVideoContent {
...
+        return [_nativeAd getMediaAspectRatio];
```
Comment (stanley-vungle) [inline adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleNativeAd.m:183]: You need to run clang to fix the identation

**Review by stanley-vungle — APPROVED** [review]

**Review by ashinagawa — APPROVED** [review]
LGTM

**Review by YueVungle — APPROVED** [review]
LGTM 👍

**Pull request merged**